### ISSUE TITLE
Submission not working with Chrome - Wait too long to submit

### ISSUE THREAD

**Issue by balabooooo** [codalab/codalab-competitions#3503]
When I click “submit”, the button is always grey and my results have never been submitted.
There was no error after a long time (about 10 mins +).
I remember the upload failed message before, but not this time.
![image](https://github.com/user-attachments/assets/46f50cc4-9e4c-492a-b844-c7f477789c12)

1. What browser and version are you using?
Microsoft Edge version 127.0.2651.74
2. What is the URL of the problem?
https://codalab.lisn.upsaclay.fr/competitions/12424

**Comment by liangjh2001** [comment]
same problem..

**Comment by balabooooo** [comment]
@liangjh2001 Hello, have you solved this?

**Comment by Didayolo** [comment]
I can't reproduce the bug.

- What is your username?
- What is the size of the file you are trying to upload? Can you try with a light ZIP file?

**Comment by liangjh2001** [comment]
> @liangjh2001 Hello, have you solved this?

I haven't solved this problem yet

**Comment by liangjh2001** [comment]
> I can't reproduce the bug.
> 
> * What is your username?
> * What is the size of the file you are trying to upload? Can you try with a light ZIP file?

My username is:  liangyun
The zip file I tried to upload is 160kb in size. I have tried smaller files and still failed. I can't even upload files that I have uploaded before.
The url of the competition I participated in is: https://codalab.lisn.upsaclay.fr/competitions/4057#participate
Looking forward to your reply

**Comment by balabooooo** [comment]
> I can't reproduce the bug.
> 
> * What is your username?
> * What is the size of the file you are trying to upload? Can you try with a light ZIP file?

@Didayolo @liangjh2001 
Hi, there! My username is balabo and the size is 14.8 MB.
**I solve this by changing to another computer and submitting!**
But I still don't know what causes it.
I think it might have something to do with the Internet environment.

**Comment by yahooo-m** [comment]
Same problem. Anyone solved?

**Comment by balabooooo** [comment]
@yahooo-m 
I solved this by changing to another computer several days ago.
However, the new computer had the same problem after several submissions.

**Comment by yahooo-m** [comment]
> @yahooo-m I solved this by changing to another computer several days ago. However, the new computer had the same problem after several submissions.

I changed to another computer. It also did not work.

**Comment by balabooooo** [comment]
> > @yahooo-m I solved this by changing to another computer several days ago. However, the new computer had the same problem after several submissions.
> 
> I changed to another computer. It also did not work.

@yahooo-m Fine

**Comment by chenz97** [comment]
same problem here, it seems not competition-specific, i have tried several different competitions. from the task manager i found that it just didn't upload file at all.

**Comment by yahooo-m** [comment]
anyone solved?

**Comment by minhduy2202** [comment]
Can we submit the file via terminal? I cannot submit file in the web version.

**Comment by baharay** [comment]
I see the following error in the console:

scripts.js:178 [Deprecation] Listener added for a 'DOMNodeInserted' mutation event. Support for this event type has been removed, and this event will no longer be fired. See https://chromestatus.com/feature/5083947249172480 for more information.

The link states:
"Mutation event support will be disabled by default starting in Chrome 127, around **July 30, 2024.**"

Could this be related?

**Comment by gpxhyfzqz** [comment]
The same problem at https://codalab.lisn.upsaclay.fr/competitions/7404#participate-submit_results.
anyone solved?

**Comment by HELLORPG** [comment]
Same issue.

**Comment by yuanziqiang1995** [comment]
Still can not submit (╥﹏╥)
https://codalab.lisn.upsaclay.fr/competitions/8582#participate-submit_results

**Comment by 4sin30** [comment]
Same problem at https://codalab.lisn.upsaclay.fr/competitions/6937#participate-submit_results

**Comment by 4sin30** [comment]
> Same problem at https://codalab.lisn.upsaclay.fr/competitions/6937#participate-submit_results

Using Firefox instead of Edge or Chorme solved the problem for me!

**Comment by gpxhyfzqz** [comment]
Thank you very much. I was also successful.


> > Same problem at https://codalab.lisn.upsaclay.fr/competitions/6937#participate-submit_results
> 
> Using Firefox instead of Edge or Chorme solved the problem for me!

**Comment by Didayolo** [comment]
I am restarting the service in case it can resolve the issue.

**Comment by balazstolnai** [comment]
It seems like it is caused by the deprecation of mutation events in Chrome. It still runs on Firefox, it only gives a warning. 
For now, it is also possible to re-enable the deprecated mutation events in chrome://flags/ and edge://flags/ (But I am not sure what downsides it may have)

**Comment by molu-fhv** [comment]
I’m experiencing the same issue - nothing happens when I click the submit button. I've tried different browsers (Firefox, Chrome, Opera, and Edge) as well as multiple laptops, but the problem persists. Does anybody have a solution to that?

**Comment by molu-fhv** [comment]
It seems to be working for me now. My solution was to increase the ZIP compression level. Initially, I had it set to 0%, but after raising the compression (resulting in smaller files), the submission system started working fine!

**Comment by embar111** [comment]
> > Same problem at https://codalab.lisn.upsaclay.fr/competitions/6937#participate-submit_results
> 
> Using Firefox instead of Edge or Chorme solved the problem for me!
Thank you very much,I have submitted.

**Comment by WesLee88524** [comment]
> It seems like it is caused by the deprecation of mutation events in Chrome. It still runs on Firefox, it only gives a warning. For now, it is also possible to re-enable the deprecated mutation events in chrome://flags/ and edge://flags/ (But I am not sure what downsides it may have)

Work for me!

**Comment by YunLi-AI** [comment]
> It seems like it is caused by the deprecation of mutation events in Chrome. It still runs on Firefox, it only gives a warning. For now, it is also possible to re-enable the deprecated mutation events in chrome://flags/ and edge://flags/ (But I am not sure what downsides it may have)

it work for me ,thanks,

**Comment by Didayolo** [comment]
The problematic code may be here:

https://github.com/codalab/codalab-competitions/blob/c7ef5e281af201c62896d9e5c8ccc1011caff979/codalab/apps/web/static/lib/js/knockout-2.2.1.debug.js#L279

**Comment by Didayolo** [comment]
@ObadaS

**Comment by ObadaS** [comment]
> The problematic code may be here:
> 
> https://github.com/codalab/codalab-competitions/blob/c7ef5e281af201c62896d9e5c8ccc1011caff979/codalab/apps/web/static/lib/js/knockout-2.2.1.debug.js#L279

After some more investigation, it seems that the problem is coming from another library.

https://github.com/codalab/django-s3direct/blob/b0b4ef267fc2d66c939766a5a622a6b889264d70/s3direct/static/s3direct/js/scripts.js#L242

**Comment by Didayolo** [comment]
Should be resolved now. Feel free to report here if you are still facing the issue.

**Comment by MSiam** [comment]
@Didayolo I am trying to submit to the Codalab competition: https://codalab.lisn.upsaclay.fr/competitions/15094 
MeVIS Referring Video Segmentation. I have tried Chrome and Firefox and it still doesnt work. I have tried to downgrade Firefox to version 1.28.0 it worked once but then stopped working once I restarted the firefox which is quite awkward. I have a deadline in 1 week though so I would really appreciate the help to fix it ASAP.

I have the same behaviour: When I click “submit”, the button is always grey and my results have never been submitted.
There was no error after a long time (about 10 mins +).

I tried to reinstall the old version again but still not working, ensured there is no automatic updates for the old Firefox version but still not working.
